website/docs/r/logz_monitor.html.markdown

```diff
@@ -0,0 +1,122 @@
+
+* `location` - (Required) The Azure Region where the logz Monitor should exist. Changing this forces a new logz Monitor to be created.
+
+* `plan_data` - (Required) A `plan_data` block as defined below.
+
+* `user_info` - (Required) A `user_info` block as defined below.
+
+---
+
+* `company_name` - (Optional) Name of the Logz organization. Changing this forces a new logz Monitor to be created.
+
+* `enterprise_app_id` - (Optional) The ID of the enterprise_app. Changing this forces a new logz Monitor to be created.
+
+~> **NOTE** Please follow [Set up Logz.io single sign-on](https://docs.microsoft.com/en-us/azure/partner-solutions/logzio/setup-sso) to create the ID of the enterprise_app.
+
+* `enabled` - (Optional) Flag specifying if the resource monitoring is enabled or disabled. Possible values are "true" and "false" is allowed.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the logz Monitor.
+
+---
+
+An `plan_data` block exports the following:
+
+* `billing_cycle` - (Required) Different billing cycles. Possible values are `MONTHLY` or `WEEKLY`. Changing this forces a new logz Monitor to be created.
+
+* `effective_date` - (Required) Date when plan was applied. Changing this forces a new logz Monitor to be created.
+
+* `plan_id` - (Required) Plan id as published by Logz. Possible values are `100gb14days`. Changing this forces a new logz Monitor to be created.
+
+* `usage_type` - (Required) Different usage type. Possible values are `PAYG` or `COMMITTED`. Changing this forces a new logz Monitor to be created.
+
+---
+
+An `user_info` block exports the following:
+
+* `email_address` - (Required) Email of the user used by Logz for contacting them if needed. Changing this forces a new logz Monitor to be created.
+
+~> **NOTE** If you use the Azure CLI to authenticate to Azure, the Email of your Azure account needs to be granted the admin permission in your Logz.io account. Otherwise, you may not be able to delete this resource. There is no such limitation for the Service Principal authentication.
+
+* `first_name` - (Required) First Name of the user. Changing this forces a new logz Monitor to be created.
+
+* `last_name` - (Required) Last Name of the user. Changing this forces a new logz Monitor to be created.
+
+* `phone_number` - (Required) Phone number of the user used by Logz for contacting them if needed. Changing this forces a new logz Monitor to be created.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the logz Monitor.
+
+* `single_sign_on_url` - The single sign on url associated with the logz organization of this logz Monitor.
+
+* `logz_organization_id` - The ID associated with the logz organization of this logz Monitor.
+
+* `liftr_resource_category` - The category of the resource.
+
+* `liftr_resource_preference` - The priority of the resource.
```

Comment:
what does liftr mean? it is in the property name but not the description?

Reply:
liftr is the name of an internal project used by the Logz.io service. Since these 2 properties are useless for the Terraform users, I have removed these 2 properties. Thanks.